mne_bids_pipeline/tests/configs/config_ds000117.py

```diff
@@ -20,6 +20,8 @@
 mf_reference_run = "02"
 mf_cal_fname = bids_root + "/derivatives/meg_derivatives/sss_cal.dat"
 mf_ctc_fname = bids_root + "/derivatives/meg_derivatives/ct_sparse.fif"
+mf_int_order = 7
+mf_ext_order = 4
```

Comment:
So comparing PR to dev

1. https://output.circle-artifacts.com/output/job/4a27513d-fd31-4174-aa37-5fb4c54e3508/artifacts/0/site/examples/ds000117/sub-01_ses-meg_task-facerecognition_report.html#Decoding__full-epochs
2. https://mne.tools/mne-bids-pipeline/stable/examples/ds000117/sub-01_ses-meg_task-facerecognition_report.html#Decoding__full-epochs

decoding performance went down. So I want to try tweaking this so that it hopefully doesn't hurt. I suspect it's the internal order being too small but maybe we'll need to decrease `ext_order` as well...
```suggestion
mf_int_order = 9
mf_ext_order = 4
```